src/mavedb/view_models/target_sequence.py

```diff
@@ -13,7 +13,8 @@ class TargetSequenceBase(BaseModel):
     sequence_type: str
     sequence: str
     label: Optional[str]
-    reference: ReferenceGenome
+    # Without this one targetGene.targetSequence?.taxonomy.organismName in front end doesn't work
+    taxonomy: Taxonomy
```

Comment:
Since Taxonomy is a required property of these target sequences and the alembic migration script is a data migration that fills in taxonomies for each target via `id`, we will just want to make sure that data migration is up to date for all score sets in the db before running the migration.

Alternatively, we could do something like
```
op.execute("""UPDATE target_sequences SET taxonomy_id = 0 WHERE taxonomy_id is NULL""")
```
as the last step of the migration, just to ensure any residually empty target sequences are filled with the empty Taxonomy.

Reply:
Target sequence will throw a validating error and leads to Search page doesn't work if taxonomy is null or empty. I think it's a good idea to prevent any new data that doesn't have taxonomy in the future.

Reply:
Target sequence will throw a validating error and lead to Search page doesn't work if taxonomy is null or empty. I think it's a good idea to prevent any new data that doesn't have taxonomy in the future.